roles/installer/tasks/main.yml

```diff
@@ -36,3 +36,6 @@
   register: foreman_installer_run
   changed_when: foreman_installer_run.rc == 2
   failed_when: foreman_installer_run.rc not in [0, 2]
+  environment:
+    LC_ALL: "{{ foreman_installer_locale | default('en_US.UTF-8') }}"
+    LANG: "{{ foreman_installer_locale | default('en_US.UTF-8') }}"
```

Comment:
somehow, this freaks out the installer:
```
The LANG environment variable should not be set to C
```
probably, because there is no `en_US.UTF-8` in the default stream8 container, and it falls back to C

Reply:
`dnf install glibc-langpack-en` should do it (only on EL8!)

Reply:
Interesting, when I check the langpacks on Rocky, ```glibc-all-langpacks``` is already present :-)

Reply:
I think we can just add it as a prep step in the molecule playbook, as it's not something needed on "normal" installs

Reply:
No, a 'normal' ISO installation, makes sense that it's missing in minimal containers. I'll add it and update the PR

Reply:
I think it's best to add it as a task to the ```installer``` role, do you agree?

Reply:
I am torn (as I don't exactly know how the langpack packages work), like what happens if the user has "all" installed and we make the role install "en"?

Reply:
Maybe we should not make a change there then outside of the CI/CD scope, most (if not all) OS installers already take care of this (I haven't seen any issues with this so far, and I have deployed _a lot_ of Foremen on CentOS7, CentOS8 and Rocky8. Same goes for Satellite).